<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompressionSettings</class>
 <widget class="QGroupBox" name="CompressionSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="rarLbl">
       <property name="text">
        <string>rar path:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="rarPathLayout">
       <item>
        <widget class="QLineEdit" name="rarPathLE">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>full path of rar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rarPathButton">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>select unrar</string>
         </property>
         <property name="text">
          <string> ... </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="genNameCB">
       <property name="toolTip">
        <string>generate a random name for each archive</string>
       </property>
       <property name="text">
        <string>generate random name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="nameLengthSB">
       <property name="toolTip">
        <string>length of the random archive name</string>
       </property>
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <layout class="QHBoxLayout" name="splitLayout">
       <item>
        <widget class="QCheckBox" name="splitCB">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>split archive with volume size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="splitSB">
         <property name="toolTip">
          <string>volume size</string>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="genPassCB">
       <property name="toolTip">
        <string>generate a random password for each archive</string>
       </property>
       <property name="text">
        <string>generate random password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="passLengthSB">
       <property name="toolTip">
        <string>length of the random archive password</string>
       </property>
       <property name="minimum">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3">
      <layout class="QHBoxLayout" name="fixedPassLayout">
       <item>
        <widget class="QCheckBox" name="fixedPassCB">
         <property name="toolTip">
          <string>use a fixeed password for all the archives</string>
         </property>
         <property name="text">
          <string>use a fixed password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fixedPassLE">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fixedPassButton">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>generate a random password</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/genKey.png</normaloff>:/icons/genKey.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="redundancyCB">
       <property name="toolTip">
        <string>WinRar -rr option</string>
       </property>
       <property name="text">
        <string>add recovery records (%): </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="redundancySB"/>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="lockCB">
         <property name="toolTip">
          <string>Winrar -k option</string>
         </property>
         <property name="text">
          <string>lock archive</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="sfvCB">
         <property name="toolTip">
          <string>create an sfv file</string>
         </property>
         <property name="text">
          <string>create sfv</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="levelLbl">
         <property name="text">
          <string>Compression level:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="levelSB">
         <property name="toolTip">
          <string>level must be between 0 and 5</string>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
